sample-compose/.gitignore

```diff
@@ -14,6 +14,7 @@
 bin/
 gen/
 out/
+build/
```

Comment:
We already ignored the `build` folder in each template and sample project. This is duplicated action 🙏 

![image](https://user-images.githubusercontent.com/16315358/203016621-12a9c99a-b306-437c-98cf-a477409184d1.png)

![image](https://user-images.githubusercontent.com/16315358/203016949-d52a5749-21c7-4d52-ac9f-7734619fca6c.png)

Reply:
@chornerman In this case, I think we should add the missing gitignore file for each `buildSrc` module instead. We should not put all git ignore config at the root file.

![image](https://user-images.githubusercontent.com/16315358/203018746-5bba667d-f322-4531-a2b2-3aea97a1bf32.png)

Reply:
I see, let me update 🚀

Reply:
@chornerman Yes, please, also for the old Rx template project

<img width="710" alt="image" src="https://user-images.githubusercontent.com/16315358/203019764-6617f1b2-70ac-4f4f-8429-a694921cbbb8.png">

Reply:
@luongvo Done 🚀